src/states/elder/mod.rs

```diff
@@ -690,12 +693,23 @@ impl Elder {
         }
     }
 
-    fn send_section_info_ack(&mut self, sec_info: SectionInfo) {
+    fn handle_ack_message(
+        &mut self,
+        sec_info: SectionInfo,
+        _src: XorName,
+        dst: XorName,
+    ) -> Result<(), RoutingError> {
+        if self.our_prefix().matches(&dst) {
```

Comment:
prefix probably don't need to match, i.e we may get an ack for the section we were before splitting.